README.md

```diff
@@ -1 +1,2 @@
 # React-boilerplate
```

Comment:
please add a section "## Deployment" where you will write how to deploy this app